packages/oscal-react-library/src/components/oscal-utils/OSCALLinkUtils.ts

```diff
@@ -1,7 +1,10 @@
 import type { BackMatter } from "@easydynamics/oscal-types";
 import { BackMatterLookup } from "./OSCALBackMatterUtils";
 
-export function getAbsoluteUrl(href: string | undefined, parentUrl: string): string | undefined {
+export function getAbsoluteUrl(
+  href: string | undefined,
+  parentUrl: string | undefined
```

Comment:
This could be an interface as well.

Reply:
I'm also wondering do we favor using `parentUrl?: string` over `parentUrl: string | undefined`? It may make sense to write it as the former in the case that we receive an empty string.

Reply:
> This could be an interface as well.

Do we gain much by doing so? The one below was created as components that call this function are passed props that match its arguments. This is not the case where we call `getAbsoluteUrl`. Not saying that I won't do it, I just am wondering why we would. 

> I'm also wondering do we favor using parentUrl?: string over parentUrl: string | undefined? It may make sense to write it as the former in the case that we receive an empty string.

I personally think that *something* should always be passed to the function. Even if it is `""`. All places we call this currently we put something there. 

I think calling a function called `getAbsoluteUrl` somewhat implies there is likely a `parentUrl` to care about. That is different from `resolveLinkHref` where you don't need a parent url to resolve a link.